### PR TITLE
Add re.sonny.Workbench

### PR DESCRIPTION
**Pull request by sonnyp**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by sonnyp** [comment]
bot, build re.sonny.Workbench

**Comment by sonnyp** [comment]
bot, build re.sonny.Workbench

File: re.sonny.Workbench.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  "modules": [
+    {
+      "name": "libadwaita",
+      "buildsystem": "meson",
+      "config-opts": ["-Dvapi=false", "-Dexamples=false", "-Dtests=false"],
+      "cleanup": ["/include", "/lib/pkgconfig"],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libadwaita/1.0/libadwaita-1.0.0.alpha.4.tar.xz",
```
Comment (barthalion) [inline re.sonny.Workbench.json:22]: 1.0.2 is out, is there something wrong with it?

Comment (barthalion) [reply re.sonny.Workbench.json:22]: It's not a blocker, please take a look at this after the merge.

Comment (sonnyp) [reply re.sonny.Workbench.json:22]: requires a newer gtk version

**Pull request closed without merging**